generators/vsi/templates/terraform/scripts/build_master.sh

```diff
@@ -20,3 +20,38 @@ mvn clean install
 mvn clean install
 target/liberty/wlp/bin/server package defaultServer --archive="{{deployment.name}}" --include=minify
 {{/has}}
+{{#has deployment.language 'GO'}}
+# get project name from manifest.yml
+export project_name="$(cat manifest.yml | grep -w "GOPACKAGENAME :" | sed 's/    GOPACKAGENAME : //')"
```

Comment:
Line 25 (this line) uses export, while line 27 does not use export. Is there a specific reason for this? We should probably keep these two consistent if possible